MoltenVK/MoltenVK/API/vk_mvk_moltenvk.h

```diff
@@ -611,6 +611,7 @@ typedef struct {
 	VkBool32 placementHeaps;					/**< If true, MTLHeap objects support placement of resources. */
 	VkDeviceSize pushConstantSizeAlignment;		/**< The alignment used internally when allocating memory for push constants. Must be PoT. */
 	uint32_t maxTextureLayers;					/**< The maximum number of layers in an array texture. */
+    uint32_t subgroupSize;			            /**< The number of threads in a wavefront. */
```

Comment:
"Wavefront" is a term specific to Direct3D and to AMD cards. This should probably use the Metal equivalent term "SIMD-group."